### ISSUE TITLE
Polymer test suite problem with flow_polymer

### ISSUE THREAD

**Issue by flikka** [OPM/opm-data-legacy#76]
Hi!

Anybody that knows why the running of the dataset at https://github.com/OPM/opm-data/tree/master/polymer_test_suite/simple2D fails? 

See https://ci.opm-project.org/job/polymer-test-simple-2d/

```
flow_polymer: /var/lib/jenkins/workspace/opm-parser-install/opm/parser/eclipse/EclipseState/Tables/SimpleTable.cpp:66: const std::vector<double>& Opm::SimpleTable::getColumn(size_t) const: Assertion `colIdx < static_cast<size_t>(m_columns.size())' failed.
Aborted
```

Perhaps @GitPaean have a clue?

**Comment by GitPaean** [comment]
Something similar happened before and fixed. I will take a look the coming Monday.

**Comment by GitPaean** [comment]
Should be fixed by OPM/opm-polymer#154 now.
